src/components/incentives/MeritIncentivesTooltipContent.tsx

```diff
@@ -41,6 +41,19 @@ export const MeritIncentivesTooltipContent = ({
         </Link>
       </Typography>
 
+      <Typography variant="caption" color="text.secondary" mb={3}>
+        <Trans>Rewards can be claimed through the</Trans>{' '}
```

Comment:
Could you please update text here to:

`Merit Program rewards are claimed through the`